### ISSUE TITLE
Display model validation errors

### ISSUE THREAD

**Issue by ain** [galtzo-floss/seed_migration#24]
Currently, if I `Model.create(data)`, seeding falls through without much knowledge about its success. Validation errors are silently ignored. It'd be good to either have an option for it or have validation errors shown by default if it fails.

**Comment by pjambet** [comment]
That's a good point, a feature like this would be interesting but I sadly don't think I'll time to work on this any time soon.

I'll keep this issue updated if I have time to work on this, and as usual I'd be more than happy to merge a PR that implements this feature :)

**Comment by ain** [comment]
Yeah, I know. There's a lot going on and TBH I had already forgotten about the thing. 

Let's label as enhancement and keep it in the backlog. I might send you a PR as I have to test things anyway.

**Comment by pjambet** [comment]
Sounds good!
